src/routes/rss.nim

```diff
@@ -105,9 +105,9 @@ proc createRssRouter*(cfg: Config) =
         of "search": initQuery(params(request), name=name)
         else: Query(fromUser: @[name])
 
-      var key = @"name" & "/" & @"tab"
+      var key = "tab/" & @"name" & "/" & @"tab" & "/"
```

Comment:
For consistency, I think it makes sense to put the tab name in front here, so we have e.g. `rss:tweets:username`, `rss:media:username`, `rss:search:username`.

```suggestion
      var key = @"tab" & ":" & @"name" & ":"
```